javascript/ios-web-test.js

```diff
@@ -22,6 +22,7 @@ const desiredCaps = {
   deviceGroup:        'KOBITON', 
   deviceName:         deviceName,
   platformName:       'iOS'
+  platformVersion:    '*'
```

Comment:
```suggestion
  platformName:       'iOS',
  platformVersion:    '*'
```
Hi @sehrgut , I think you miss ` ,`